### PR TITLE
[CHANGES] in Tests summary (table)

### PR DESCRIPTION
**Pull request by iariasleon**
Now in the new format, the summary is presented as a table

See a example in https://jirapdi.tid.es/secure/attachment/912737/regressiom_test.txt

### CODE REVIEW

File: test/acceptance/behave/environment.py [hunk 1]
```diff
@@ -125,6 +125,20 @@ def init_feature():
     scenarios_status = {"untested": 0, "skipped": 0, "passed": 0, "failed": 0}
 
 
+def __add_chars(text, before, after, char_to_add=" "):
+    """
+    append chars before or/and after in a text
+    :param text: text to modify
+    :param before: number of blanks added to the beginning of the text
```
Comment (fgalan) [inline test/acceptance/behave/environment.py]: number of blanks -> number of chars

(You mention in char_to_add comment that blank is the default, but not the general case)

Comment (iariasleon) [inline test/acceptance/behave/environment.py]: Yes, initially in this function only blank spaces are appended, now, it is used with any character.
Fixed in  	b5998b6

**Comment by jcanonav** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM

**Pull request merged**